src/Barman.java

```diff
@@ -0,0 +1,32 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class Barman {
+    public Drink createDrink() {
```

Comment:
This method should accept 6 params. Follow exercise instructions!

Reply:
Due to this change my code looks poor and much worse than before. But the task is the task, teacher.